evm_arithmetization/src/cpu/kernel/tests/core/access_lists.rs

```diff
@@ -94,89 +226,102 @@ fn test_insert_accessed_addresses() -> Result<()> {
 
 #[test]
 fn test_insert_accessed_storage_keys() -> Result<()> {
+    let init_access_lists = KERNEL.global_labels["init_access_lists"];
+
+    // Test for address already in list.
+    let initial_stack = vec![0xdeadbeefu32.into()];
+    let mut interpreter = Interpreter::<F>::new_with_kernel(init_access_lists, initial_stack);
+    interpreter.run()?;
+
     let insert_accessed_storage_keys = KERNEL.global_labels["insert_accessed_storage_keys"];
 
     let retaddr = 0xdeadbeefu32.into();
     let mut rng = thread_rng();
-    let n = rng.gen_range(1..10);
-    let storage_keys = (0..n)
+    let n = 10;
+    let mut storage_keys = (0..n)
         .map(|_| (rng.gen::<Address>(), U256(rng.gen()), U256(rng.gen())))
         .collect::<HashSet<_>>()
         .into_iter()
         .collect::<Vec<(Address, U256, U256)>>();
+    // Storage keys must be sorted
+    storage_keys.sort();
     let storage_key_in_list = storage_keys[rng.gen_range(0..n)];
```

Comment:
Unused?

Reply:
I was necessary when setting the memory manually.